docs/5g-on-helium/mobile-poc.mdx

```diff
@@ -74,82 +64,260 @@ brief interruptions.
 
 ---
 
-## Stage 1: Genesis Rewards
+## Speed Tests and Quality of Service
 
-:::info Reward Targets
+:::caution Coming Soon
 
-During Genesis, 100M Mobile tokens are allocated to be issued per day. This 24-hour Reward Period is
-expressed in blocks, and aligns as close to the Reward Period time range as possible, but may not
-line up _exactly_ on 24-hour intervals.
-
-Given this slight differentiation, daily rewards may not equate to an exact 100M MOBILE. Check out
-this
-[Example Reward Transaction](https://explorer.helium.com/txns/0n4TO_2W14rrsGt-nxBTotiHFCFukkO04QoXIxaOtAY)
-for reference.
+Speed Test metrics are not yet included in Rewards calculations.
 
 :::
 
-### Genesis Phase 1: Time Period Triggered Rewards
+A 5G Hotspot will automatically perform a Speed Test approximately every 12 hours with the results
+being averaged by the MOBILE PoC Oracle.
 
-> - Reward Period is approximately 1440 blocks and closely aligns with 00:00 UTC to 23:59 UTC.
-> - Reward submission occurs approx 17 hours later, once a day, around 17:00 UTC.
+Soon, the MOBILE PoC Oracle will include these Speed Test metrics in the Rewards calculations for
+each 5G Hotspot.
 
-At this Milestone, to qualify for the Genesis Rewards in a Reward Period, a CBRS Small Cell Radio
-must:
+The "Moving Average", also known as a "Rolling Average", is the average of at least `2` but no more
+than `6` most recent Speed Test results for each download, upload, and latency for a given 5G
+Hotspot.
 
-- Be connected to a 5G Hotspot.
-- Be properly installed.
-  - Be registered with the SAS (Outdoor Radios Only).
-- Broadcast at least 1 "heartbeat" per Reward Period.
+Additionally, the Moving Average is reset following a Speed Test gap of `>12 hours` and a new series
+of Speed Test results must be collected before a Moving Average can be calculated.
 
-### Genesis Phase 2: Time Period Triggered Rewards with Minimum Delay
+### Tiered Speed Test
 
-:::info
+In response to community feedback and further exploration by Chris from DeWiPulse for his
+[analysis of and commentary on public Internet data in the United States](https://dewipulse.com/connectivity-where-it-matters/)
+Speed Test results are categorized into one of four Tiers.
+
+| Speed Test Tier | Reward Multiplier | Requirement (speeds in Mbps)                                   |
```

Comment:
Optionally for consistency how the values are displayed in the table:
Requirement (speeds in Mbps) > Requirement (speeds in Mbps, latency in ms) and remove ms from table